hknweb/candidate/templates/candidate/index.html

```diff
@@ -1,24 +1,34 @@
 {% extends "base.html" %}
 {% load static %}
 
-<!-- I just made this to have a place to put the officer stuff.
-     Feel free to delete this page later, as long as you still
-     keep the mechanism for making officer challenge requests. -->
+{% block header %}
+<link rel="stylesheet" href="{% static "candidate/styles.css" %}">
+{% endblock %}
 
 {% block title %}Candidate Portal{% endblock %}
 
-{% block header %}
-  <!--<link rel='stylesheet' href="http://unpkg.com/fullcalendar/dist/fullcalendar.min.css" />-->
-  <!--<script src="http://unpkg.com/moment/min/moment.min.js"></script>-->
-  <!--<script src="http://unpkg.com/jquery/dist/jquery.slim.min.js"></script>-->
-  <!--<script src="http://unpkg.com/fullcalendar/dist/fullcalendar.min.js"></script>-->
+{% block heading %}
+Candidate Portal
 {% endblock %}
 
 {% block content %}
-<div class="center measure-very-wide pv4">
-    <h1 class="font-title f3 f2-ns tc">Candidate Portal</h1>
+<div class="parent">
+
+    <div style="float:right; width: 300px">
+        <section id='agenda' style="background-color: #EEEEEE; margin: 0.5%; padding-left: 20px; padding-right: 20px; padding-top: 5px; padding-bottom: 10px; width: 150">
```

Comment:
This is a bit messy; in particular, you could shorten the padding to a single line with something like:
```css
padding: 5px 20px 10px
```
for 5px top, 20px horizontal, and 10px bottom padding.
(You might also want to move this into a .css file or something.)

Reply:
I've done that since